### PR TITLE
Supression

### PR DESCRIPTION
**Pull request by marwin1991**

### CODE REVIEW

**Comment by marwin1991** [comment]
@angrymeir when you will have some time can you take a look at this change ? :)

**Comment by marwin1991** [comment]
@angrymeir

**Comment by marwin1991** [comment]
@angrymeir Will you find a time this week to look at this change?

**Review by angrymeir — CHANGES_REQUESTED** [review]
Wow! This looks super awesome - great work!!! Really, thats some next level shit 🎉

File: secscanner2junit/sast.py [hunk 1]
```diff
@@ -1,14 +1,19 @@
-from .parser import Parser
-from junit_xml import TestSuite, TestCase
 from collections import defaultdict
 from random import randrange
 
+from junit_xml import TestSuite, TestCase
+
+# is it correct import changed from .parser ? I've get error ImportError: attempted relative import with no known parent package
+from secscanner2junit.config import Config
```
Comment (marwin1991) [inline secscanner2junit/sast.py:7]: @angrymeir is it corrent import?

File: tests/test_config.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import unittest
+
+from secscanner2junit.config import get_config, Config, Suppression
+
+
+class TestConfig(unittest.TestCase):
+
+    def test_get_config_no_file(self):
+        # given:
+        input_config_path = "resources/test_config/test_get_config_no_file/ss2ju-config.yml"
```
[the remark this answers is not in the document]
Comment (marwin1991) [reply tests/test_config.py:10]: I have added empty `test_get_config_no_file` dir, and this test is checking if when file does not exist, the program should return empty config

File: tests/test_config.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    def test_get_config(self):
+        # given:
+        input_config_path = "resources/test_config/test_get_config/ss2ju-config.yml"
+
+        # when:
+        config = get_config(input_config_path)
+
+        # then:
+        expected = Config([Suppression('cwe', '2555'), Suppression('find_sec_bugs_type', 'SPRING_ENDPOINT')])
+        assert expected == config
```
[the remark this answers is not in the document]
Comment (marwin1991) [reply tests/test_config.py]: Yes, you are rigth :)

**Review by angrymeir — APPROVED** [review]

**Pull request merged**